web/src/components/table/data-table.tsx

```diff
@@ -239,11 +246,22 @@ export function DataTable<TData extends object, TValue>({
           </Table>
         </div>
       </div>
+      {/* ensures consistent spacing across tables with and without DataTableToolbar, h-12 given fixed height of DataTableToolbar */}
+      <div className="hidden lg:block lg:h-12"></div>
```

Comment:
Yes, this makes up for the height of the data table toolbar when not present. A not nice but intuitive way to write this would be: 
 
`{isTableToolbarNotSet && <div className="lg:h-12"></div>}`